<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainBox</class>
 <widget class="QWidget" name="MainBox">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>355</width>
    <height>162</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="RS232_widget" name="rs232_widget" native="true"/>
     </item>
     <item>
      <widget class="Sound_widget" name="sound_widget" native="true"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Generator_Curve" name="generator_widget" native="true"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Generator_Curve</class>
   <extends>QWidget</extends>
   <header>generator_curve.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>RS232_widget</class>
   <extends>QWidget</extends>
   <header>rs232_widget.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Sound_widget</class>
   <extends>QWidget</extends>
   <header>sound_widget.hpp</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
